modules/dnn/src/layers/elementwise_layers.cpp

```diff
@@ -1129,17 +1187,47 @@ struct MishFunctor : public BaseDefaultFunctor<MishFunctor>
                backendId == DNN_BACKEND_CANN;
     }
 
+    // It is needed by tryQuantize. We can remove this in 5.x.
     inline float calculate(float x) const
     {
-        // Use fast approximation introduced in https://github.com/opencv/opencv/pull/17200
-        if (x >= 8.f)
-        {
-            return x;
-        }
+        x *= (x > -36.73f ? 1.f : 0.f);
+        float y = std::exp(-x);
+        return x * (1 + 2 * y) / (1 + 2 * y + 2 * y * y);
+    }
 
-        float eX = exp(x);
-        float n = (eX + 2.f) * eX;
-        return (x * n) / (n + 2.f);
+    void apply(const float* srcptr, float* dstptr, int stripeStart, int len, size_t planeSize, int cn0, int cn1) const {
+        CV_UNUSED(stripeStart);
+        for (int cn = cn0; cn < cn1; cn++, srcptr += planeSize, dstptr += planeSize) {
+            int i = 0;
+#if (CV_SIMD || CV_SIMD_SCALABLE)
+            v_float32 threshold = vx_setall_f32(-36.73f), one = vx_setall_f32(1.f), z = vx_setzero_f32();
```

Comment:
The same for the constant. It's used several times. I propose to introduce named constant with description and use it.

Reply:
Guarded the constant with an anonymous namespace.